pkg/util/paramtable/component_param.go

```diff
@@ -3844,6 +3845,14 @@ mix is prioritized by level: mix compactions first, then L0 compactions, then cl
 	}
 	p.CompactionTaskQueueCapacity.Init(base.mgr)
 
+	p.CompactionPreAllocateIDExpansionFactor = ParamItem{
```

Comment:
I'm not sure if stats task need this too.

Reply:
Nice catch, stats use a hardcode factor 100, I'll unify it as well.